spk/bazarr/src/dsm-control.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Package
+PACKAGE="bazarr"
+DNAME="Bazarr"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${PYTHON_DIR}/bin:${PATH}"
+BAZARR_DIR="${INSTALL_DIR}/share/bazarr"
+PID_FILE="${INSTALL_DIR}/var/bazarr.pid"
+SH_PATH=/bin/sh
+
+USER="sc-bazarr"
+
+start_daemon ()
+{
+    start-stop-daemon -S -q -m -b -N 10 -p "${PID_FILE}" -x "${SH_PATH}"  -- \
+        -c "source \"${INSTALL_DIR}/bin/settings.sh\" \
+        && pushd \"${BAZARR_DIR}\" \
+        && mkdir -p \"${BAZARR_DIR}/var\" \
+        && python bazarr.py >>\"${BAZARR_DIR}/var/bazarr.log\" 2>&1"
+}
+
+stop_daemon ()
+{
+    curl http://127.0.0.1:6767/shutdown || echo "Failed invoking smooth shutdown procedure";
```

Comment:
OK, I give you that this is pretty ugly and won't work if the port is changed which is ... bad.

An alternative would be to go with PPID mechanisms, WDYT ?

Reply:
Fixed to use PPID process lookup.